tests/MongoDB.Driver.Core.Tests/Specifications/connection-string/connection-string-spec.rst

```diff
@@ -0,0 +1,386 @@
+.. role:: javascript(code)
```

Comment:
No need to copy this file to our repo.

We only need to sync test `tests` directory itself.

Reply:
Done.